chart/kubeapps/values.yaml

```diff
@@ -1561,6 +1561,8 @@ kubeappsapis:
             ## headerPattern: namespace:^([\w-]+):\w+$
             ##
             headerPattern: ""
+          ## @param kubeappsapis.pluginConfig.core.packages.v1alpha1.forwardedHeaders Optional list of headers that should be forwarded from the incoming gRPC-Web request to the k8s API server.
```

Comment:
i think this is at `kubeappsapis.pluginConfig.resources.packages.v1alpha1.forwardedHeaders`?

Reply:
Woops, yes, I initially tried it in the core settings but thought it better here with the resources (which are required anyway).